### PR TITLE
rebase: update minikube to v1.23.0 (backport #2488 ) for release-v3.3 

### PR DESCRIPTION
**Pull request by Rakshith-R**
rebase: update minikube to v1.23.0 (backport #2488 ) for release-v3.3 

Add ability to create extra disks for hyperkit driver
Add ability to create extra disks for kvm2 driver
See-also: https://github.com/kubernetes/minikube/releases/tag/v1.23.0

Signed-off-by: Rakshith R rar@redhat.com

### CODE REVIEW

**Review by Madhu-1 — DISMISSED** [review]

**Review by yati1998 — DISMISSED** [review]

**Comment by Rakshith-R** [comment]
https://github.com/ceph/ceph-csi/runs/3532604807?check_suite_focus=true
added `cleanup: address pylint "consider-using-with" in tracevol.py` to fix lint-extra failures. PTAL

**Review by Madhu-1 — DISMISSED** [review]

File: scripts/rook.sh [hunk 1]
```diff
@@ -75,6 +84,12 @@ function teardown_rook() {
 	kubectl delete -f "${ROOK_URL}/toolbox.yaml"
 	kubectl delete -f "${ROOK_URL}/cluster-test.yaml"
 	kubectl delete -f "${ROOK_URL}/operator.yaml"
+	ROOK_MAJOR=$(rook_version 1)
```
Comment (humblec) [inline scripts/rook.sh:88]: not sure we need to have an explicit delete based on the version, but thats fine too

**Review by humblec — DISMISSED** [review]

**Comment by Rakshith-R** [comment]
```

rook-ceph-mds-myfs-a-7578ccc4cc-vbh66           0/1     CrashLoopBackOff   7          13m
rook-ceph-mds-myfs-b-7865bb7f87-gwm6x           0/1     CrashLoopBackOff   7          13m
rook-ceph-mgr-a-54d74555c9-h45cn                1/1     Running            0          15m
rook-ceph-mon-a-8f87b454c-wck6r                 0/1     CrashLoopBackOff   9          16m
```

https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/rest/organizations/jenkins/pipelines/upgrade-tests-cephfs/runs/3040/nodes/96/log/?start=0
Adding commit to disable liveness probes

**Comment by Rakshith-R** [comment]
> /retest ci/centos/upgrade-tests-cephfs

https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/upgrade-tests-cephfs/detail/upgrade-tests-cephfs/3042/pipeline
`etcdserver: request timed out`

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Pull request merged**